src/radical/pilot/agent/agent_0.py

```diff
@@ -340,75 +349,74 @@ def _start_services(self):
         If a `./services` file exist, reserve a compute node and run that file
         there as bash script.
         '''
+        self._log.info('Starting the agent services')
+        service_descriptions = self._cfg.services
```

Comment:
```python
if not service_descriptions:
     return
```

Reply:
and move `self._log.info('Starting the agent services')` after it

Reply:
and remove docstring

Reply:
fixed